processing/src/main/java/io/druid/query/aggregation/JavaScriptAggregatorFactory.java

```diff
@@ -40,7 +40,7 @@
 import java.util.Comparator;
 import java.util.List;
 
-public class JavaScriptAggregatorFactory implements AggregatorFactory
+public class JavaScriptAggregatorFactory extends AggregatorFactory
```

Comment:
I guess, what's the desired behavior for merging JavaScriptAggregator-created columns that have the same name but different fnCombines?